src/Dom/Browser.Dom.fsproj

```diff
@@ -2,8 +2,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <PackageId>Fable.Browser.Dom</PackageId>
-    <Version>2.7.0</Version>
-    <PackageVersion>2.7.0</PackageVersion>
+    <Version>2.8.0</Version>
+    <PackageVersion>2.8.0</PackageVersion>
```

Comment:
You don't need to bump the version here. This is automatically done by the build script :)

Reply:
Got it! 👍